### PR TITLE
Restored and actualized README.md file

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Restores and actualizes README.md file.

The PR is supposed to be reviewed in a commit-by-commit manner.

### What issues does this PR fix or reference?
https://github.com/eclipse/che-jwtproxy/issues/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+Run tests in project with:
+```bash
+go test ./...
+```
+
+## Build Linux Binary file locally
+
+Build Binary file locally with:
+```
+go build -o jwtproxy cmd/jwtproxy/main.go
```
Comment (mshaposhnik) [inline README.md]: Need to put note about CGO_ENABLED=0  somewhere

File: README.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+      # Registerable key server type and options used to fetch
+      # public keys for verifying signatures
+      key_server:
+         type: <string|nil>
+         options: <map[string]interface{}>
+
+      # Registerable type and options where we track used nonces
+      nonce_storage:
+        type: <string|nil>
+        options: <map[string]interface{}>
```
Comment (mshaposhnik) [inline README.md]: pls add excludes examle

File: README.md (hunk 1 above)
Comment (sleshchenko) [reply README.md]: I checked and it works for me without `CGO_ENABLED=0`. Could you elaborate on why it is needed? When may it not work without it?

File: README.md (hunk 2 above)
Comment (sleshchenko) [reply README.md]: Thanks for reminding. I'll add

File: README.md (hunk 1 above)
Comment (mshaposhnik) [reply README.md]: uhm.. last time i tried it won't start without it..

File: README.md (hunk 2 above)
Comment (sleshchenko) [reply README.md]: Added here https://github.com/eclipse/che-jwtproxy/pull/8/files#diff-04c6e90faac2675aa89e2176d2eec7d8R185

**Pull request merged**